packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -83,7 +83,6 @@ export const ParticipantListWithHeading = (props: {
                 {...options}
                 {...{ hidePersonaDetails: !!options?.text }}
                 dataProvider={onFetchAvatarPersonaData}
-                /* @conditional-compile-remove(raise-hand) */
                 allowActiveBorder={true}
```

Comment:
nit: if this is always true now, can probably remove this prop completely

Reply:
we need to keep this as other places that use the persona don't pass this in. so we see a bunch of style changes as a result of these being missing